### PR TITLE
Replace StatusObject with underlying JSON type

### PR DESCRIPTION
**Pull request by michael-loki**

### CODE REVIEW

File: llarp/handlers/exit.cpp [hunk 1]
```diff
@@ -51,9 +51,9 @@ namespace llarp
       util::StatusObject exitsObj{};
       for(const auto &item : m_ActiveExits)
       {
-        exitsObj.Put(item.first.ToHex(), item.second->ExtractStatus());
+        exitsObj[item.first.ToHex()] = item.second->ExtractStatus();
```
Comment (majestrate) [inline llarp/handlers/exit.cpp]: while you're here it ideally should be `ToString()` not `ToHex()` so it reports the snode address

File: llarp/service/endpoint_state.cpp [hunk 2]
```diff
@@ -116,31 +116,40 @@ namespace llarp
...
+      std::transform(
+          m_SNodeSessions.begin(), m_SNodeSessions.end(),
+          std::back_inserter(obj["snodeSessions"]),
+          [](const auto& item) { return item.second.first->ExtractStatus(); });
 
       util::StatusObject sessionObj{};
 
       for(const auto& item : m_Sessions)
       {
         std::string k = item.first.ToHex();
```
Comment (majestrate) [inline llarp/service/endpoint_state.cpp]: also `ToString()`

**Comment by michael-loki** [comment]
Will do.

File: llarp/service/endpoint_state.cpp (hunk 2 above)
Comment (michael-loki) [reply llarp/service/endpoint_state.cpp]: `No member named 'ToString' in 'llarp::AlignedBuffer<16>'`

Comment (majestrate) [reply llarp/service/endpoint_state.cpp]: okay nvm there then

**Comment by michael-loki** [comment]
@majestrate fixed up most of the instances of `ToHex`

**Pull request merged**